mppi_node:
  ros__parameters:
    horizon: 20
    num_samples: 1000
    lambda_weight: 1.0
    dt: 0.1
    model_path: "weights/dynamics_model.pt"
    control_frequency: 10.0
